packages/hydrogen/vitest.setup.ts

```diff
@@ -3,3 +3,14 @@ import matchers from '@testing-library/jest-dom/matchers';
 import {expect} from 'vitest';
 
 expect.extend(matchers);
+
+const perfkit = document.createElement('script');
+perfkit.setAttribute('data-application', 'hydrogen');
+perfkit.setAttribute('data-shop-id', '1');
+perfkit.setAttribute('data-storefront-id', '0');
+perfkit.setAttribute('data-monorail-region', 'global');
+perfkit.setAttribute('data-spa-mode', 'true');
+
+Object.defineProperty(document, 'currentScript', {
+  value: perfkit,
+});
```

Comment:
This is the only place where I can define a read only document variable. If anyone knows of another way that isn't applying to the all test, let me know.

Reply:
How does this really work in tests? Does it download the perfkit min js file?
Can't we just mock `window.PerfKit` or something like that? 🤔 

Otherwise, it's not a big issue having it here I guess, as long as we keep a comment on what is this doing for the test.